notebook/services/contents/fileio.py

```diff
@@ -130,8 +185,12 @@ def open(self, os_path, *args, **kwargs):
     def atomic_writing(self, os_path, *args, **kwargs):
         """wrapper around atomic_writing that turns permission errors to 403"""
```

Comment:
This docstring should be updated now that the method may not actually do atomic_writing depending on configuration.

Also, ideally we'd rename the method to reflect what it now does. But I'm not sure whether the method counts as public API; if it does, we shouldn't rename it.